overlord/fdestate/fdemgr.go

```diff
@@ -181,6 +188,87 @@ func (m *unlockedStateManager) Unlock() (relock func()) {
 	return func() {}
 }
 
+// EncryptedDisk implements backend.EncryptedDisk
+type EncryptedDisk struct {
```

Comment:
does it need to be exported? for tests you can add the following in `export_test.go`:
```
type EncryptedDisk = encryptedDisk
```

Reply:
It needs ti be used in tests of `./overlord/fdestate/backend/`, but it is defined it `./overlord/fdestate/`.

Reply:
I can maybe just reimplement it.